src/mappings/entities/block/mapChainUpgrade.ts

```diff
@@ -29,6 +30,11 @@ export default async (substrateEvent: SubstrateEvent): Promise<void> => {
       logger.info(`Transaction version was not changed for the chain upgrade`);
     }
 
+    if (specVersion >= 7000000 && specVersion <= 7000999) {
+      logger.info('v7 upgrade detected - migrating claim tickers to asset IDs');
+      await v7ClaimIdHandler(substrateEvent);
+    }
+
```

Comment:
we should avoid doing as many actions at chain upgrade. If we just fix the id of claims to always use assetId, this won't be needed

Reply:
The SQL doesn't work either since the process gets sandboxed in SQ.

I should be able to update just IDs using the ORM though.

Reply:
That won't be necessary, we can handle fixing the ID of the claims entity to always use assetId from scope and voila, we don't need to do anything